guides/common/modules/proc_increasing-the-logging-levels-to-help-with-debugging.adoc

```diff
@@ -29,17 +29,17 @@ ifndef::foreman-deb[]
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 # hammer admin logging --list
------------|-------------------------------------|-------------------------------------
-COMPONENT  | AUTO-DETECTED BY EXISTENCE OF       | DESTINATIONS
------------|-------------------------------------|-------------------------------------
-dhcpd      | /etc/dhcp/dhcpd.conf                | syslog /var/log/dhcpd-debug.log
-postgresql | /var/lib/pgsql/data/postgresql.conf | syslog /var/lib/pgsql/data/pg_log/
-proxy      | /etc/foreman-proxy/settings.yml     | /var/log/foreman-proxy/proxy.log
-qpidd      | /etc/qpid/qpidd.conf                | syslog
-rails      | /etc/foreman/settings.yaml          | /var/log/foreman/production.log
-tomcat     | /etc/candlepin/candlepin.conf       | /var/log/candlepin/ /var/log/tomcat/
-virt-who   | /etc/sysconfig/virt-who             | syslog
------------|-------------------------------------|-------------------------------------
+-----------|---------------------------------------|-------------------------------------
```

Comment:
The alignment looks odd here. I think I'll need to revert the additional spaces.